src/Adapter/Order/AbstractOrderHandler.php

```diff
@@ -243,6 +245,35 @@ protected function updateSpecificPrice(
         $specificPrice->add();
     }
 
+    /**
+     * @param Number $priceTaxIncluded
+     * @param Number $priceTaxExcluded
+     * @param Order $order
+     * @param Product $product
+     *
+     * @return Number
```

Comment:
```suggestion
     * @return \PrestaShop\Decimal\Number
```

Reply:
All for the sake of PHPStan that is only enabled on the develop branch 😅

Reply:
I'm kidding, I'm gonna fix it to ease the merge
But seriously it's a HUGE pain that we need to use the fully qualified class name, this is just gonna make our code uglier and uglier, reduce the motivation to use this library, and prevent us from using our IDE instinctively..
In my opinion it's PHPStan that should adapt to this use case, with a configuration or something, not us..

Reply:
Open an issue on PHPStan repository :p